### PR TITLE
Remaing changes for the update POA work

### LINKED ISSUE #1085: Styleguide component | Introduction to layouts
As a developer, I want clear and consistent guideline for how the layout of all Caseflow apps should look like. 

## Acceptance Criteria

- Verify that the description copy is below the title 
- Verify that layouts has the following subsections. **_Note: These sections don't have to be completed but it's good to have the title placeholder._**
  1. Navigation bar 
  1. User dropdown menu
  1. Main content area
  1. App canvas
  1. Footer

## Copy

Any Caseflow app should always follow the same layout and structure shown here. At the top of the application is the Navigation Bar. Next, sitting on the canvass of the application is the main content area. This is where the title of the page, instructions for the user, and detailed interaction should be placed. The primary and secondary actions sit below the main content area. Finally, the Footer should go at the bottom of the app. 

We try to be consistent in this layout across all our applications so that our interface is predictable and familiar to our user. The consistency also helps us reuse common code in Caseflow Commons.

### PR DESCRIPTION
**Pull request by nweinmeister**
Resolves #1085

### Description
The update POA functionality should be limited to only VLJ support team members and if there's not already a POA (for now). See this thread for discussion on how to determine if user is in VLJ support: https://dsva.slack.com/archives/CJL810329/p1628875432220100

### Acceptance Criteria
- User cannot see the Update POA button if the user is not in the VLJ Support team role

BEFORE
Screen Shot 2021-08-16 at 2 11 17 PM

AFTER
Screen Shot 2021-08-16 at 2 10 05 PM

Screen Shot 2021-08-16 at 2 49 56 PM

### CODE REVIEW

File: spec/feature/queue/unrecognized_appellant_spec.rb [hunk 1]
```diff
@@ -140,11 +140,12 @@
     before { FeatureToggle.enable!(:edit_unrecognized_appellant_poa) }
     after { FeatureToggle.disable!(:edit_unrecognized_appellant_poa) }
 
-    it "should show the edit information button if there's a POA already" do
+    it "should NOT show the edit information button if there's a POA already" do
```
Comment (Sjones352) [inline spec/feature/queue/unrecognized_appellant_spec.rb:143]: Just a nit "should not instead of capitalizing NOT

**Comment by Sjones352** [comment]
Can you add a screenshot since you made some frontend changes

File: spec/feature/queue/unrecognized_appellant_spec.rb (hunk 1 above)
Comment (nweinmeister) [reply spec/feature/queue/unrecognized_appellant_spec.rb:143]: I can do that!

**Review by Sjones352 — APPROVED** [review]
LGTM

**Review by meckila-navapbc — APPROVED** [review]

**Pull request merged**